### PR TITLE
[Rule Tuning] Q2 Linux DR Tuning - Part 2

### PR DESCRIPTION
**Pull request by Aegrah**
## Summary
Several Linux DR tunings

### CODE REVIEW

**Comment by protectionsmachine** [comment]
## Rule: Tuning - Guidelines

These guidelines serve as a reminder set of considerations when tuning an existing rule.

### Documentation and Context

- [ ] Detailed description of the suggested changes.
- [ ] Provide example JSON data or screenshots.
- [ ] Provide evidence of reducing benign events mistakenly identified as threats (False Positives).
- [ ] Provide evidence of enhancing detection of true threats that were previously missed (False Negatives).
- [ ] Provide evidence of optimizing resource consumption and execution time of detection rules (Performance).
- [ ] Provide evidence of specific environment factors influencing customized rule tuning (Contextual Tuning).
- [ ] Provide evidence of improvements made by modifying sensitivity by changing alert triggering thresholds (Threshold Adjustments).
- [ ] Provide evidence of refining rules to better detect deviations from typical behavior (Behavioral Tuning).
- [ ] Provide evidence of improvements of adjusting rules based on time-based patterns (Temporal Tuning).
- [ ] Provide reasoning of adjusting priority or severity levels of alerts (Severity Tuning).
- [ ] Provide evidence of improving quality integrity of our data used by detection rules (Data Quality).
- [ ] Ensure the tuning includes necessary updates to the release documentation and versioning.

### Rule Metadata Checks

- [ ] `updated_date` matches the date of tuning PR merged.
- [ ] `min_stack_version` should support the widest stack versions.
- [ ] `name` and `description` should be descriptive and not include typos.
- [ ] `query` should be inclusive, not overly exclusive. Review to ensure the original intent of the rule is maintained.

### Testing and Validation

- [ ] Validate that the tuned rule's performance is satisfactory and does not negatively impact the stack.
- [ ] Ensure that the tuned rule has a low false positive rate.

File: rules/linux/defense_evasion_sus_utility_executed_via_tmux_or_screen.toml [hunk 1]
```diff
@@ -29,26 +29,23 @@ tags = [
...
 timestamp_override = "event.ingested"
 type = "eql"
-
 query = '''
 process where host.os.type == "linux" and event.type == "start" and event.action in ("exec", "exec_event") and 
-process.parent.name in ("screen", "tmux") and process.name : (
-  "nmap", "nc", "ncat", "netcat", "socat", "nc.openbsd", "ngrok", "ping", "java", "python*", "php*", "perl", "ruby",
-  "lua*", "openssl", "telnet", "awk", "wget", "curl", "id"
-  )
+process.parent.name in ("screen", "tmux") and process.name like (
```
Comment (terrancedejesus) [inline rules/linux/defense_evasion_sus_utility_executed_via_tmux_or_screen.toml:34]: @Aegrah did we test the use of this in Kibana EQL? I assume it's built from the same grammar so should still be feasible as endpoint, but just wanted to be sure as they are different in some capabilities.

File: rules/linux/discovery_esxi_software_via_find.toml [hunk 2]
```diff
@@ -59,23 +59,22 @@ tags = [
...
 timestamp_override = "event.ingested"
 type = "eql"
-
 query = '''
-process where host.os.type == "linux" and event.type == "start" and event.action in ("exec", "exec_event", "executed", "process_started")
- and process.name == "find" and process.args : ("/etc/vmware/*", "/usr/lib/vmware/*", "/vmfs/*")
+process where host.os.type == "linux" and event.type == "start" and
+event.action in ("exec", "exec_event", "executed", "process_started") and process.name == "find" and
+process.args : ("/etc/vmware/*", "/usr/lib/vmware/*", "/vmfs/*") and 
+not process.parent.executable == "/usr/lib/vmware/viewagent/bin/uninstall_viewagent.sh"
```
Comment (terrancedejesus) [inline rules/linux/discovery_esxi_software_via_find.toml:66]: Is there an install script we need to ignore for vmware as well that may use the same find process?

File: rules/linux/discovery_kernel_module_enumeration.toml [hunk 3]
```diff
@@ -66,30 +66,33 @@ event.category:process and host.os.type:linux and event.type:start and event.act
  (process.name:(lsmod or modinfo)) or 
  (process.name:kmod and process.args:list) or 
  (process.name:depmod and process.args:(--all or -a))
-) and not process.parent.name:(mkinitramfs or cryptroot or framebuffer or dracut or jem or thin-provisioning-tools
-or readykernel or lvm2 or vz-start or iscsi or mdadm or ovalprobes or bcache or plymouth or dkms or overlayroot or 
-weak-modules or zfs)
+) and
+not (
+  process.parent.name:(
```
Comment (terrancedejesus) [inline rules/linux/discovery_kernel_module_enumeration.toml:71]: can we do process.parent.name like () here since these are all specific strings?

Comment (Aegrah) [reply rules/linux/discovery_kernel_module_enumeration.toml:71]: It's KQL, so unfortunately no

File: rules/linux/defense_evasion_sus_utility_executed_via_tmux_or_screen.toml (hunk 1 above)
Comment (Aegrah) [reply rules/linux/defense_evasion_sus_utility_executed_via_tmux_or_screen.toml:34]: I just checked, and it works. But good call!

File: rules/linux/discovery_esxi_software_via_find.toml (hunk 2 above)
Comment (Aegrah) [reply rules/linux/discovery_esxi_software_via_find.toml:66]: Looking at telemetry I think this should be enough. This rule in general is not noisy; it was just 1 FP.

**Review by shashank-elastic — APPROVED** [review]

**Pull request merged**